### DISCUSSION TITLE
Matomo and wiki.js

### DISCUSSION THREAD

**Discussion by 40foot** [requarks/wiki#3230] [Help / Questions] [answered]
I self-host wiki.js under UNRAID as a docker container. Under Administration - Analytics there can be Matomo (also a self-hosted docker) activated, but nothing happens. Can anyone direct me as to how this can be accomplished? Oh, if someone could just tell me, if this module isn't usable, where to put the JavaScript-Tracking-Code from Matomo, can't find a header.php or something similar.

**Comment by NGPixel** [comment]
Did you enter the settings for Matomo and did you enable the module? The js tracking code will automatically be included in your page.

**Comment by 40foot** [comment]
thx for your reply. my settings are as follows:Am 17.03.2021 um 00:57 schrieb Nicolas Giard ***@***.***>:Did you enter the settings for Matomo and did you enable the module? The js tracking code will automatically be included in your page.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by 40foot** [comment, marked as answer]
IT WORKS !!! - just restarted the docker, now it gathers everything. Thanks for your attention! I’m lovin’ it… (now I have to find out, why I can’t upload any images anymore, will start a new thread on this).
